oio_rest/tests/test_db_helpers.py

```diff
@@ -456,7 +456,8 @@ def test_get_relation_names_default(self):
                 "tilknyttedeorganisationer",
                 "tilknyttedeitsystemer",
                 "tilknyttedeinteressefaellesskaber",
-                "tilknyttedepersoner"
+                "tilknyttedepersoner",
+                "tilknyttedefunktioner"
```

Comment:
Although this is mostly a matter of taste, terminating commas have the advantage of not breaking `blame` whenever you add a new element:

```suggestion
                "tilknyttedefunktioner",
```

Reply:
I agree, but I went with fewer (unrelated) diff lines and local consistency (the rest of the structure has many lists without terminating commas).